atomicapp/nulecule/base.py

```diff
@@ -302,6 +303,7 @@ def stop(self, provider_key=None, dryrun=False):
         provider_key, provider = self.get_provider(provider_key, dryrun)
         provider.artifacts = self.rendered_artifacts.get(provider_key, [])
         provider.init()
+        provider.ignore_errors = True
```

Comment:
However, there's no usecase where ``ignore_errors`` is gonna come as ``False`` here.